packages/core/admin/package.json

```diff
@@ -107,7 +107,6 @@
     "prop-types": "^15.7.2",
     "qs": "6.10.1",
     "react": "^17.0.2",
-    "react-copy-to-clipboard": "^5.1.0",
```

Comment:
I don't think you can remove it as it's still used [elsewhere](https://github.com/strapi/strapi/blob/f2450b9298e32eb9467fb54c24d22084705db074/packages/core/admin/admin/src/pages/SettingsPage/pages/Users/components/MagicLink/MagicLinkWrapper.js#L6) in the admin. I supposed it's because of browser compatibility concerns

Reply:
oh wow, thanks that was a huge mistake

Reply:
No worries that's why we have reviews :)